README.md

```diff
@@ -94,15 +92,14 @@ The guidelines described below are based on:
 
   *Why?*: Keeps consistency with AngularJS 1.x style guidelines.
 
-  ### Example
-
-  **CORRECT**
   ```
+  /* CORRECT */
```

Comment:
Here is what I was attempting to refine.

![screen shot 2016-02-15 at 9 35 20 am](https://cloud.githubusercontent.com/assets/8420490/13052971/9526cf1a-d3c7-11e5-903c-c531db0da2b9.png)

Headings and even the bold text have a bunch of extra padding. Add all of that padding up and figure in the number of examples and I was concerned that the sections will get excessively long. 

**Example** - On a 3k screen, I have to scroll 7 times to make it through the entire "Directives & Components" section.

> Perhaps the solution is to find a balance between "sub-sections" & a more obvious comment or tighter "correct/wrong" headings.

I'll create a throw away branch with just the Directives & Components section repeated in a few variations and you can pick the one your prefer. I'll apply that back to this branch and sprinkle a bit of rebase love on it.